lib/constants.js

```diff
@@ -22,21 +22,21 @@ const ISSUE_TRANSITION_ACTION_NAME = {
     START_PROGRESS: 'Start Progress',
     RESOLVE: 'Resolve',
     CLOSE: 'Close',
-} // action can take to transition for issue in workflow
+}) // action can take to transition for issue in workflow
 
-const ISSUE_LINK_TYPE_NAME = { RFC_FRD: 'RFC-RFD' }
+const ISSUE_LINK_TYPE_NAME = Object.freeze({ RFC_FRD: 'RFC-RFD' })
 
-const ENV = { BUILD: 'build', DEV: 'dev', DLVR: 'dlvr', TEST: 'test', PROD: 'prod' }
-const JIRA_TARGET_ENV = { DLVR: 'DLVR', TEST: 'TEST', PROD: 'PROD' }
-const REVIEWER_ENV_MAP = new Map([
-    [JIRA_TARGET_ENV.DLVR, 'Developer'],
-    [JIRA_TARGET_ENV.TEST, 'IIT'],
-    [JIRA_TARGET_ENV.PROD, 'Business'],
-])
-const DB_ACTION = { BACKUP: 'backup', RECOVERY: 'recovery' }
+const ENV = Object.freeze({ BUILD: 'build', DEV: 'dev', DLVR: 'dlvr', TEST: 'test', PROD: 'prod' })
+const JIRA_TARGET_ENV = Object.freeze({ DLVR: 'DLVR', TEST: 'TEST', PROD: 'PROD' })
+const JIRA_ENV_REVIEWER = [
+    Object.freeze({ ENV: 'DLVR', REVIEWER: 'Developer' }),
+    Object.freeze({ ENV: 'TEST', REVIEWER: 'IIT' }),
+    Object.freeze({ ENV: 'DLVR', REVIEWER: 'Business' }),
+]
```

Comment:
The objects in the array are frozen. Q: Is the array frozen? What would happen if I did:
JIRA_ENV_REVIEWER[0] = 'bob';

Reply:
corrected